pkg/operator/mon/mon.go

```diff
@@ -199,14 +208,19 @@ func (c *Cluster) initMonServices(size int) ([]*monConfig, error) {
 		if err != nil {
```

Comment:
we don't have a need to create the service when hostNetwork is enabled, right?

Reply:
Mhh good point. No we actually don't need the services for `hostNetwork` because we don't use the DNS and we simply instead of DNS or the "generated" Cluster IPs, the selected/assigned node IP addresses are used.

Reply:
The HostIP would be changed, if we have several k8s workers. 
It means the address is still not stable as stable as service.
When pod is restarting, it might cause some issues.

Reply:
@travisn Right now, when a mon is replaced/failovered the service IP will change too, so there is no difference. But I'm working on changing this behavior to be more stable.

Reply:
@timclu are you referring to the scenario of failing over a mon to another host? In that case it is expected that it would get a new ip as @galexrt mentioned. If you're referring to the HostIP changing for another reason such as using DHCP, it would be up to the network admin to configure the machines to have a stable HostIP when using host networking.